src/main/resources/templates/editPassword.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+    <title>Edit Password</title>
+</head>
+<body>
+<div id="nav-placeholder"></div>
+<div class="login-container">
+    <form action="#" th:action="@{/editPassword}" th:object="${editPassword}" method="post"
+          class="container d-flex flex-column flex-md-col col-sm-12 col-md-6 col-lg-4">
+        <h3>Edit Your Password</h3><br>
+        <div class="form-group">
+            <label>Enter Your Password:</label>
+            <input type="password" class="form-control"
+                   name="oldPassword" id="oldPassword"
+                   required="" placeholder="required"/>
+        </div>
+        <div class="password_match" data-msg="Password doesn't match.">
+            <div class="password form-group">
+                <label>Enter New Password:</label>
+                <input type="password" name="newPassword"
+                       id="password" class="form-control"
+                       required="" placeholder="required"
+                       pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
+                       title="8 Characts: 1 Upper, 1 Lower, 1 Digit, 1 Special Characts. Must Contain!!!">
+            </div>
+            <div class="passwordConfirm form-group">
+                <label>Confirm New Password:</label>
+                <input type="password" id="passwordConfirm"
+                       class="form-control" placeholder="required"
+                       pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
+                       required="" title="Must match password">
+            </div>
+            <div>
+                <label>
+                    <input type="checkbox" onclick="togglePasswordVisibility()">
+                    <label class="form-check-lable">
+                        Show Password's
```

Comment:
is this supposed to be plural? should this show both at once? i only know this togglePasswordVisibility with this eye icon - maybe look into that (bet bootstrap has some)

Reply:
yes it should show all so you can correct typo mistakes by comparing them side by side and also correct the old Password if there were then typo mistakes of approving

Reply:
done